storage/entities/transaction.js

```diff
@@ -684,6 +492,39 @@ class Transaction extends BaseEntity {
 
 		return filters;
 	}
+
+	static _sanitizeCreateData(data) {
+		const transactions = Array.isArray(data)
+			? _.cloneDeep(data)
+			: [_.cloneDeep(data)];
+
+		transactions.forEach(transaction => {
```

Comment:
instead of `forEach` map would be better?

Reply:
I reckon `forEach` is clearer when mutating the object

Reply:
I guess the point is not to mutate and return the new object =)

Reply:
But it's already cloning the object in the line before. So it's not mutating the passed param.